library/network/src/modules/NetworkInterfaces.rb

```diff
@@ -744,6 +744,15 @@ def add_device(device, ifcfg)
       @Devices[devtype][device] = ifcfg
     end
 
+    # Device configuration files are matched against this regex
+    #
+    # The regex defines files which should not be parsed (e.g. ifcfg-eth0.bak)
+    #
+    # @return [Regexp] regexp describing ignored configurations
+    def ignore_confs_regex
+      /(.bak|.orig|.rpmnew|.rpmorig|.rpmsave|-range|~|.old|.scpmbackup)$/
```

Comment:
But *why* use these and not others? Should it be synced with another project (wicked, ...)? A URL link to the other project's definition would be nice.

`man ifcfg` on 42.3 tells me
> Configuration filenames that contain the following blacklisted extensions, will be ignored by wicked:
>       ~ .old .bak .orig .scpmbackup .rpmnew .rpmsave .rpmorig